src/components/modal/modal.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../styles/variables';
+
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: $color-bg-dark-1;
+}
+
+.modal {
+  position: absolute;
+  top: 40px;
+  right: 40px;
+  bottom: 40px;
+  left: 40px;
+  background-color: $color-bg-dark-1;
+}
+
+div {
+  color: white;
```

Comment:
All of these selectors must be scoped to a classname in BEM style, as this project isn't yet using any CSS modules so these styles will affect the whole app (the light theme for example font colour has changed to white on white)

Reply:
sounds good, this is the type of comment / suggestion I am hoping to get for the styling - I've changed the styling in my latest commit but will revisit it to comply more with BEM

Reply:
@bru5 updated styling to follow BEM style in latest commit - let me know your thoughts on the new naming of the classes